product_docs/docs/biganimal/release/using_cluster/fault_injection_testing/index.mdx

```diff
@@ -68,27 +69,28 @@ p-x67kjp3fsq-a  2456382099  data     world   p-x67kjp3fsq-a-1
 p-x67kjp3fsq-c  4147262499  data     world                    
 p-x67kjp3fsq-d  3176957154  data     world   p-x67kjp3fsq-d-1
 ```
-In this example, the write leader node is **p-x67kjp3fsq-a-1**.
+In this example, the write leader node is `p-x67kjp3fsq-a-1`.
 
 
 ## Deleting a write leader node from your cluster
 
 To delete a write lead node from the cluster:
 1. Log into BigAnimal.
 2. In a separate browser window, log into your Microsoft Azure subscription.
-3. In the left navigation of BigAnimal portal, choose **Clusters**. 
-4. Choose the cluster to test fault injection with and copy the string value from the URL. The string value is located after the underscore.  
+3. In the left navigation of BigAnimal portal, select **Clusters**. 
+4. Select the cluster to test fault injection with and copy the string value from the URL. The string value is located after the underscore.  
 
      ![Delete a write lead](images/biganimal_faultinjectiontest_1.png)
 
 
-5. In your Azure subscription, paste the string into the search and prefix it with **dp-** to search for the data plane.  
-   * From the results, choose the Kubernetes service from the Azure Region that your cluster is deployed in.
+5. To search for the data plane, in your Azure subscription, paste the string into the search and prefix it with `dp-`.  
```

Comment:
This really needed to be its own step. It's not really a bullet item of the pervious step. You could have made it all one step.

Also, I recommend just using the number 1. for steps so you don't have to reorder if you decide to add steps. The steps come out in order if they are all numbered 1.